src/activityPubPlugin.ts

```diff
@@ -35,12 +35,17 @@ export const activityPubPlugin = (eleventyConfig: EleventyConfig, {
 				}
 			],
 			summary,
-			icon: avatar && {
+			icon: avatar ? {
```

Comment:
Is there a particular reason you've changed this? `&&` should evaluate to the same if `avatar` is `undefined`

Reply:
It's because `avatar` (incoming arg) is a string type, so when it's falsey, it cannot be assigned to `icon`. I could update the `icon` type instead to allow `string`, but felt like that's not really what you want here. Let me know how you'd like to proceed though :)

<img width="781" alt="Screenshot 2023-01-02 at 8 14 39 AM" src="https://user-images.githubusercontent.com/490968/210256371-0ab6851a-15a5-45d0-b4a8-6ded6b541700.png">

Reply:
had to go one step further because prettier + lint somehow ended up with mixed tabs and spaces which failed lint 🤦🏾